platforms/js/TextClip.hx

```diff
@@ -403,8 +403,10 @@ class TextClip extends NativeWidgetClip {
 
 		advanceCorrection = untyped (getAdvancedWidthsCorrection(tm, style, eochi, egchi, egb)-getAdvancedWidthsCorrection(tm, style, bochi, bgchi, bgb)) / UPM * style.fontSize;
 
```

Comment:
Please, add comments about the idea

Reply:
This was here for ages, but ok, I will remember what this was about.

Reply:
UPD: this was done for same goal scriptingFixSuffix invented for, but for some reason did not work recently. I will remove it and see if working.